events/message.go

```diff
@@ -111,6 +111,10 @@ type Bounce struct {
 	TransmissionID  string            `json:"transmission_id"`
 }
 
+func (b *Bounce) EventType() string {
```

Comment:
What is the purpose of this. It seems like it is either valuable on all the events or more likely redudnant with 
    `func (e EventCommon) EventType() string { return e.Type }`